Source/UI/NewUI/BillingForm/ViewModel/DefaultBillingFormViewModel.swift

```diff
@@ -6,9 +6,9 @@ import Checkout
  */
 
 final class DefaultBillingFormViewModel: BillingFormViewModel {
-
+    
     // MARK: - Properties
-
+    
```

Comment:
Is SwiftLint running on these files?

Reply:
yes, and passed on CI too

Reply:
@harry-brown-cko can trailing whitespace be added without getting fixed on build command?

Reply:
Frames swiftlint is a lot less strict than 3DS and Checkout. We should improve this at some point, swiftlint currently does very little on Frames.

We should be remove trailing whitespace in any new code and as we find it but its not enforced on CI

Reply:
Next iOS meeting, we will discuss it.